### PR TITLE
MINOR: Include number of members in group coordinator messages

### PR DESCRIPTION
**Pull request by ijuma**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala [hunk 1]
```diff
@@ -561,7 +561,8 @@ class GroupCoordinator(val brokerId: Int,
 
             // if this is the leader, then we can attempt to persist state and transition to stable
             if (group.isLeader(memberId)) {
-              info(s"Assignment received from leader for group ${group.groupId} for generation ${group.generationId}")
+              info(s"Assignment received from leader for group ${group.groupId} for generation ${group.generationId}. " +
+                s"The group has ${group.allMembers.size} members, ${group.allStaticMembers.size} of which are static.")
```
Comment (chia7712) [inline core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala]: Could you replace `${group.allMembers.size}` by `${group.size}`?

**Review by chia7712 — APPROVED** [review]
LGTM

**Review by hachikuji — APPROVED** [review]
LGTM also

File: core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala (hunk 1 above)
Comment (ijuma) [reply core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala]: Done.

**Comment by ijuma** [comment]
Cherry-picked to the 2.8 branch as well.

**Pull request merged**